ansible/playbooks/AdoptOpenJDK_Windows_Playbook/roles/NVidia_Cuda_Toolkit/tasks/main.yml

```diff
@@ -39,7 +39,7 @@
   tags: NVidia_Cuda_Toolkit
 
 - name: Install NVidia CUDA toolkit - Windows 10
-  win_shell: C:\temp\cuda_9.1.85_win10_network.exe -s compiler_9.1 nvml_dev_9.1
+  win_shell: C:\temp\cuda_9.1.85_win10_network.exe -s nvcc_9.1 nvml_dev_9.1
```

Comment:
`nvcc_9.1` is the correct sub-package name according to https://docs.nvidia.com/cuda/archive/9.1/cuda-installation-guide-microsoft-windows/index.html.